File: Factorial.cpp
#include<iostream>
using namespace std;
int factorial(int num){
    long long result=1;
    for(int i=1;i<=num;i++){
        result = result*i;
        
    }
    return result;
}

int main(){
    int num;
    cout<<"Enter the Number"<<endl;
    cin>>num;
    long long result = factorial(num);
    cout<<"Factorial of "<<num<<" is "<<result;
}

File: Factorial.java
import java.util.Scanner;

class Factorial{
    public static void main(String args[]){
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the Number : ");
        int num = sc.nextInt();

        if(num == 1 || num == 0){
            System.out.println("The factorial of "+num+" is "+num);
            return;
        }
        int i = 1;
        int fact = 1;
        while(i <= num){
            fact = fact * i;
            i++;
        }
        System.out.println("The factorial of "+num+" is "+fact);
    }
}